src/lib.rs

```diff
@@ -45,3 +45,38 @@ impl fmt::Display for NonSortedIntegers {
 }
 
 impl Error for NonSortedIntegers {}
+
+/// An [`IntoIterator`] blanket implementation that provides extra methods for [`RoaringBitmap`]
+/// and [`RoaringTreemap`]
+///
+/// This trait is parameterized by a sealed trait and cannot be implemented for types outside
+/// of this crate
+pub trait IterExt<T>: IntoIterator<Item = T>
```

Comment:
Maybe we could find a better name for this trait, as it is a trait that provides set operations on multiple sets at the same time, maybe `MultiOps` or something like that? What do you think?

Reply:
Sure. We could also create a Roaring prelude that includes iter extensions, RoaringBitmap, RoaringTreemap

Reply:
Hum... not sure we have enough types and traits for now for that to be interesting, but maybe later with more types.